d2l-localize-behavior.js

```diff
@@ -1,5 +1,5 @@
 import '@polymer/polymer/polymer-legacy.js';
-import { AppLocalizeBehavior } from '@polymer/app-localize-behavior/app-localize-behavior.js';
+import { AppLocalizeBehavior } from './app-localize-behavior.js';
```

Comment:
If we import this `as D2LAppLocalizeBehavior`, would that make those other PRs in the FRAs unnecessary?  Then there should be no collisions if they import it as well

Reply:
These aren't global variables, so this part shouldn't be a problem no matter what. The problem in the other FRAs will be if we switch them back to using the normal `app-localize-behavior` and not the fork, then they could still end up with 2 copies of `intl-messageformat`.